src/context/SpeciesContext.js

```diff
@@ -31,14 +33,18 @@ export function SpeciesProvider(props) {
   // EVENT HANDLERS
 
   const loadSpeciesList = async (abortController) => {
+    setIsLoading(true);
     const species = await api.getSpecies(abortController);
     setSpeciesList(species);
+    setIsLoading(false);
   };
 
   // only used by Species dropdown
   const onChange = async (text) => {
+    // setIsLoading(true);
     console.log('on change:"', text, '"');
```

Comment:
Instead of using console, you can use `log.debug` if it's for debugging purposes. We want to limit the use of console.log.